.github/workflows/lint.yml

```
@@ -22,4 +22,4 @@ jobs:
         uses: actionsx/prettier@v2
         with:
           # prettier CLI arguments.
-          args: --check '**/*.md'
+          args: --check '**/*.{md,yml,yaml}' '.prettierrc' 'vcpkg.json'
```

We can change this to just `.` if we're expanding it to check the entire project, it will go through all directories where it makes sense (skipping things in .prettierignore)

```suggestion
          args: --check .
```